src/uu/users/src/users.rs

```diff
@@ -10,59 +10,44 @@
 // Allow dead code here in order to keep all fields, constants here, for consistency.
 #![allow(dead_code)]
 
-extern crate getopts;
+extern crate clap;
+#[macro_use]
 extern crate uucore;
 
 use uucore::utmpx::*;
 
-use getopts::Options;
+use clap::{App, Arg};
 
-static NAME: &str = "users";
+static ABOUT: &str = "Output who is currently logged in according to FILE.";
```

Comment:
Nit; attempting more consistency between uutils and slightly better? grammar/clarity.

```suggestion
static ABOUT: &str = "Display who is currently logged in, according to FILE.";
```